pkg/kudoctl/cmd/init/prereqs.go

```diff
@@ -155,12 +169,18 @@ func PrereqManifests(opts Options) ([]string, error) {
 
 // Prereq returns the slice of prerequisite objects for KUDO
 func Prereq(opts Options) []runtime.Object {
-	ns := namespace(opts.Namespace)
-	svc := serviceAccount(opts)
-	rbac := roleBinding(opts)
-	secret := webhookSecret(opts)
+	var prereqs []runtime.Object
+
+	if opts.Namespace == defaultns {
```

Comment:
please add a comment here that we only want to create the default ns, all others are expected to exist. Thanks!

Reply:
Fixed